packages/themes/src/variables.css

```diff
@@ -107,7 +108,7 @@
   }
 }
 /* High contrast custom selection */
-/* *::selection {
+*::selection {
```

Comment:
This needs to be scoped or it escapes our app into other peoples apps

Reply:
moved it in reset stylesheet which is scoped - might make more sense

Reply:
going to tag in @hwkr to see if thats the best spot, but it does work

Reply:
I think it makes sense to be in our reset :)